source/index.html.md

```diff
@@ -145,3 +145,113 @@ sugar_g | Float | The average amount of sugar consumed by all students in the sp
 sodium_g | Float | The average amount of sodium (salt) consumed by all students in the specified subset in g.
 saturated_fat_g | Float | The average amount of saturated fat consumed by all students in the specified subset in g.
 unsaturated_fat_g | Float | The average amount of unsaturated fat consumed by all students in the specified subset in g.
+
+
+
+
+
+
+
+# National School Register API
+
+## Summary by location
+
+```ruby
+require 'nuonic'
+
+api = Nuonic::APIClient.authorize!('my-api-key')
+api.nuonic.get
+```
+
+```python
+import nuonic
+
+api = nuonic.authorize('my-api-key')
+api.nuonic.get()
+```
+
+```shell
+curl "http://api.nuonic.com.au/kids_health/summary_by_location"
+  -H "Authorization: my-api-key"
+```
+
+```javascript
+const nuonic = require('nuonic');
+
+let api = nuonic.authorize('my-api-key');
+let nuonic = api.nuonic.get();
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "government_id"      : 1042,
```

Comment:
You should not have the spaces between the key and value in this response example. I know its easier to read (for you) but is not the convention for JSON output. We should align to what developers are used to, to make it easier for them not us.